### PR TITLE
Update Evolution Rock Paper Scissors

### PR DESCRIPTION
**Pull request by selynna**
Added clarification and an example

### CODE REVIEW

File: cohort_4/playbook/ACTIVITIES.md [hunk 1]
```diff
@@ -129,17 +129,16 @@ Source: Andrea
 
 Everyone starts off in the “egg” stage and has to act like eggs like being in a
 squatting position. Every “egg” will find another “egg” to play rock paper
-scissors with. Play rock paper scissors 3 times (one round) to see if either
-wins 2 out of 3. If not, play another round until someone wins a round 2 out
-of 3. The person that wins will then evolve into the next stage “monkey” and has
-to find another “monkey” to play rock paper scissors with. The “losing” person
+scissors with. Play rock paper scissors until someone wins 2 out of 3. The person that wins the rps round will then evolve into the next stage, which is “chicken,” and has
```
Comment (maxwofford) [inline cohort_4/playbook/ACTIVITIES.md]: Try wrapping this to 80 characters per line.

**Comment by maxwofford** [comment]
Thanks @ssun098! Left a comment :-)

**Comment by selynna** [comment]
@MaxWofford could you check it now? I'm not sure if I did it correctly though.

**Comment by maxwofford** [comment]
@ssun098 Thanks! Is it okay if I bundle these commits up into one?

**Comment by selynna** [comment]
Yeah, go for it!

File: cohort_4/playbook/ACTIVITIES.md [hunk 2]
```diff
@@ -129,17 +129,19 @@ Source: Andrea
...
+The “losing” person will remain an egg and find other eggs to play rock paper 
+scissors with. The stages are egg to chicken to monkey to human then to divine 
+being. In each stage, the “losing” person will de-evolve into the previous 
+stage or if an egg will remain an egg. So if you lose when you're a chicken, 
+then you become an egg again.  Once a person becomes a divine being he/she 
+leaves the game and watches.
 Continue until you have a certain number of divine beings agreed upon at the
 beginning of the game.
 
+For example, assume there are 6 people, and they all default start at the "egg" stage.  There will be 3 rounds of rock paper scissors total, so egg1 - egg2, egg3 - egg4, egg5 - egg6.  There will then be 3 winners, so chicken1 egg2 egg3 chicken4 egg5 chicken6, for example.  Then, there will be 2 rounds going on, since there are an odd number of chickens and eggs:  chicken1 - chicken6, egg2 - egg3, while chicken4 and egg5 wait for partners.  Then when people lose, an example scenario is: egg1, monkey6, chicken2, egg3, chicken4, and egg5.  This continues until there are a specific number of divine beings you decide upon.
```
Comment (zachlatta) [inline cohort_4/playbook/ACTIVITIES.md]: Can this be wrapped to the 80 character line limit?

Also, can an extra new line be created after this one to conform to our markdown style guides (linked in https://github.com/hackedu/hackedu/blob/master/CONTRIBUTING.md#style-guides)

**Comment by zachlatta** [comment]
Cherry-picked onto master in 3fe7e9d58442046ba09200ee611a8a745120935b. Safe to delete branch!

**Pull request closed without merging**